Source/dom/models/Document.js

```diff
@@ -470,3 +471,46 @@ Document.define('path', {
     })
   },
 })
+
+Document.define('colors', {
+  array: true,
+  get() {
+    if (!this._object) {
+      return []
+    }
+    const documentData = this._getMSDocumentData()
+    const colorAssets = toArray(documentData.assets().colorAssets())
+    return colorAssets.map(a => ({
+      color: Style.colorToString(a.color()),
+      name: a.name(),
```

Comment:
So thinking about it, I'm not sure it's enough to return a simple object here:
```
doc.colors[0].name = 'new name'
```
is not going to work for example.

So I think we need to create a new wrapped object.

But otherwise, all those methods look good 👍

Reply:
Ah, yup. 

I put a ColorAsset model in but then remembered gradients. Any thoughts on dealing with those 2? 

I'm Thinking 2 distinct classes would be the least confusing. Another potential option could be to abstract the internals with one class that wraps either "asset" type and exposes `gradient` and `color` properties where one always returns null/undefined - maybe with an "assetType".

Reply:
That’s the 2 options I had in mind as well, not sure what’s the best. 

I was also thinking about having `value` instead of `color` so it can be the same for all assets. 

But maybe having 2 separate classes will be more flexible for the future